### DISCUSSION TITLE
Wondering…

### DISCUSSION THREAD

**Discussion by georgie91** [lamemakes/pilot-drive#28] [General]
Have you looked into integrating with something like the iDataLink hardware? I’m not asking you to, just asking if you looked into it and what kind of response you received. Thanks!

**Comment by lamemakes** [comment]
I have not! But it could definitely be something to look into. Do you know if  they have documentation for interfacing?

**Reply by georgie91** [reply]
Not that I have found publicly. I have requested developer resources through their generic information request form. I'm not too hopeful with that route. It looks like they interface with other radios via a single data wire and an audio jack (this might just be fore steering wheel controls). I'm still looking into wiring diagrams and such the figure out which components are needed for full functionality.

**Reply by georgie91** [reply]
My guess is that it's a 4 wire can network connection.

**Reply by georgie91** [reply]
What I have found, and this isn't a great solution as a wider scale solution would cause the need for vehicle specific firmware, but it's probably more better to read/transmit the CAN data directly from/to the vehicle.

**Reply by lamemakes** [reply]
Yeah I wanted to explore some kind of like app-ish modularity so that would be a cool add-on in that regard I think.
